components/Navigator/MainNavigator.tsx

```diff
@@ -41,6 +41,9 @@ export const MainNavigator: FC = observer(() => {
             <Button variant="outline-light" size="sm" href="/agenda/0/editor">
               {t('submit_agenda')}
```

Comment:
```suggestion
             <Button variant="outline-light" size="sm" href="/session/0/editor">
               {t('submit_agenda')}
```
先创建“活动环节”，再将其提交为不同活动的“议程”让组织者来审核、排期。但为了方便人去理解，文字上可统称为“议程”。